boards/airfy-beacon/Makefile.include

```diff
@@ -6,7 +6,5 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 OPENOCD_DEBUG_ADAPTER ?= stlink
 STLINK_VERSION ?= 2
 
-PROGRAMMER = openocd
-
```

Comment:
Unrelated?

Reply:
You asked me to take care of it, I did :) Openocd is already set as default programmer in cpu/common/nrf51/Makefile.include. So this is what you were asking for, with no code duplication :)

Reply:
Ah thanks, since my change suggestion was only adding `?=` I missed this change, thanks!